apps/omstillingsstoenad-ui/src/components/felles/rhf/RHFInput.tsx

```diff
@@ -112,21 +116,25 @@ export const RHFValutaInput = ({ name, valgfri, ...rest }: RHFProps) => {
     const error: FieldError = get(errors, name) as FieldError
     const feilmelding = !!error ? t(getTransKey(error)) : undefined
 
+    const re = /^[0-9\s]+$/
+
     return (
-        <Controller
-            name={name}
-            control={control}
-            rules={{ required: !valgfri, pattern: /^\d[0-9\s]*$/ }}
-            render={({ field: { value, onChange } }) => (
-                <TextField
-                    id={name}
-                    value={value || ''}
-                    onChange={onChange}
-                    error={feilmelding}
-                    {...rest}
-                />
-            )}
-        />
+            <Controller
+                    name={name}
+                    control={control}
+                    rules={{ required: true, minLength, maxLength }}
```

Comment:
Synes kanskje det er litt dumt å fjerne pattern siden vi da mister beskrivende feilmelding (feltet må inneholde kun siffer, e.l.). Og har du lagt til tekst for feilmelding på min/max length?

Reply:
La til pattern igjen, men siden det kun er lov til å skrive inn tall og space, så vil man fort ikke få de feilmeldingene. Skulle man da heller gått for å la bruker skrive inn hva de vil og deretter se det er feil?

Fjernet min og max for valuta da det ikke er brukt.